src/client/main.cpp

```diff
@@ -41,9 +55,30 @@ static void _libhook_init() {
   } else {
     setenv("RM2FB_ACTIVE", "1", true);
   }
+
+  qImageCtor = (void (*)(void *, int, int, int))dlsym(
+      RTLD_NEXT, "_ZN6QImageC1EiiNS_6FormatE");
+  qImageCtorWithBuffer = (void (*)(
+      void *, uint8_t *, int32_t, int32_t, int32_t, int, void (*)(void *),
+      void *))dlsym(RTLD_NEXT, "_ZN6QImageC1EPhiiiNS_6FormatEPFvPvES2_");
+
   SHARED_BUF = swtfb::ipc::get_shared_buffer();
 }
 
+bool FIRST_ALLOC = true;
+void _ZN6QImageC1EiiNS_6FormatE(void *that, int x, int y, int f) {
+  if (IN_XOCHITL && x == swtfb::WIDTH && y == swtfb::HEIGHT && FIRST_ALLOC) {
+    fprintf(stderr, "REPLACING THE IMAGE with /dev/shm/xofb \n");
```

Comment:
is this the right path? isn't it placed in /dev/shm/swtfb.01 or something? you can probably delete this line print line if the info is printed elsewhere

Reply:
again, copied :)